lib/nx_signal/transforms.ex

```diff
@@ -0,0 +1,21 @@
+defmodule NxSignal.Transforms do
+  import Nx.Defn
+
+  deftransform fft_nd(tensor, opts \\ []) do
```

Comment:
This function and module are missing documentation.

If we're going to keep this module private (which is fine by me), it should have @moduledoc false

Reply:
I believe the functions here are untested as well